frontend/src/pages/modelRegistrySettings/CreateModal.scss

```diff
@@ -0,0 +1,6 @@
+// targeting specific form section title elements to override --pf-v5-c-form__section-title--FontSize, --pf-v5-c-form__section-title--FontWeight
+.form-subtitle-text {
+    color: var(--pf-v5-global--Color--200);
```

Comment:
I wonder if we can use utility classes for these instead?

```ts
import text from '@patternfly/react-styles/css/utilities/Text/text';
```

then you can use `text.color_200` in the className. Although there's no utility class for `font-size: initial` or `font-weight: initial`, you'd have to explicitly use the specific size and weight classes.

Reply:
maybe best to leave for another PR, i actually grabbed this CSS from another spot that was doing the same thing. we'd likely want to either update it there as well. i'd say we could make a custom component for this "section title/description", but there is a PF issue to add a description property to the form section.

Reply:
For posterity: the existing example of this being copied here is from https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/concepts/pipelines/content/configurePipelinesServer/ConfigurePipelinesServerModal.scss#L10